frontend/src/loadPostHogJS.tsx

```diff
@@ -67,6 +67,7 @@ export function loadPostHogJS(): void {
                     capture_copied_text: true,
                 },
                 person_profiles: 'always',
+                __preview_remote_config: true,
```

Comment:
This breaks flags - I'm working on the proper fix atm but just beware of merging

Reply:
Removed this line for now... I'm also hoping this is what caused `alerts.cy` and `onboarding.cy` to flake as well 🤞 😅

Reply:
yeah it is